scenes/game/local_game.gd

```diff
@@ -5128,10 +5172,11 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			_append_log_full(Enums.LogType.LogType_CharacterMovement, performing_player, "moves from space %s to %s." % [str(old_space), str(performing_player.arena_location)])
 			_append_log_full(Enums.LogType.LogType_CharacterMovement, performing_player, "moves %s from space %s to %s." % [performing_player.get_buddy_name(), str(old_buddy_space), str(performing_player.get_buddy_location())])
 		"take_bonus_actions":
-			var num = effect['amount']
-			performing_player.bonus_actions += num
-			performing_player.cancel_blocked_this_turn = true
-			_append_log_full(Enums.LogType.LogType_Effect, performing_player, "gains %s bonus actions!" % str(num))
+			if not performing_player.no_extra_actions_or_strike_this_turn:
+				var num = effect['amount']
+				performing_player.bonus_actions += num
+				performing_player.cancel_blocked_this_turn = true
+				_append_log_full(Enums.LogType.LogType_Effect, performing_player, "gains %s bonus actions!" % str(num))
```

Comment:
maybe this (and strike, etc.) could have a log message if the boolean is true, just saying that you cant strike/take a bonus action/etc. when a strike is already active

Reply:
I think since the cards have this text, it isn't necessary. But maybe we can add it later.